scattering/van_hove.py

```diff
@@ -214,3 +214,117 @@ def compute_partial_van_hove(trj, chunk_length=10, selection1=None, selection2=N
         g_r_t += g_r_t_frame
 
     return r, g_r_t
+def atom_conc_from_list(atom_list, trj):
+     """ 
+    Generates a dictionary of atom concentration with the key corresponding to the atoms in atom_list
+
+    Parameters
+    ----------
+    trj : mdtrj.Trajectory
+        trajectory on which partial vhf were calculated form
+    atom_list : list, optional, default = None
+        list of unique atoms. if left at default, atom_list generated from trj
+
+
+    Return
+    -------
+    mf : dictionary
+        dictionary containing concentration of atoms from atom_list
+    """
+
+    #atom_list generation
+    if atom_list = None:
+        topology = trj.topology
+        atom_list = []
+        for i in range(topology.n_atoms):
+            if topology.atom(i).name not in atom_list:
+                atom_list.append(topology.atom(i).name)
+        atom_list = sorted(set(atom_list))
+
+    atom_num_list = []
+    mf = {}
+    for i in range(len(atom_list)):
```

Comment:
Similar comment here.  For the atom count, I think we can get that by counting the items in `atoms` defined above like `atoms.count("O")`.

Reply:
Can you just loop through `atom_list` directly?
Like `for atom in atom_list`?